OBAKit/Helpers/NSKeyedUnarchiver+UnarchiveTopLevelObjectObjC.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  NSKeyedUnarchiver+UnarchiveTopLevelObjectObjC.swift
+//  OBAKit
+//
+//  Created by Alan Chu on 1/18/20.
+//  Copyright © 2020 OneBusAway. All rights reserved.
+//
+
+import Foundation
+
+extension NSKeyedUnarchiver {
+    /// Obj-C compatible method of unarchiving NSCoding objects for iOS 13. Don't use this for NSSecureCoding objects.
```

Comment:
Can you elaborate on why having two code paths—one for iOS 13 and one for iOS 11 and 12 is preferable to what exists today?

Reply:
updated code documentation to better explain this.

technically, because we still building against iOS 11, we can revert back to using the deprecated method. xcode won't complain about it unless we build against iOS 13.